site/contents/write_ups/2021/11-08-21_writeup.md

```diff
@@ -0,0 +1,119 @@
+
+- 1.6 billion trees would have to be planted to offset the pollution caused by email spam.
+- 1.5 billion trees would need to be planted to deal with annual e-commerce returns in the US alone. 
+- 231 million trees would need to be planted to deal with the pollution caused as a result of the data US citizens consumed in 2019.
+- 16 million trees would need to be planted to offset the pollution caused by the estimated 1.9 trillion yearly searches on Google. 
+
+Maybe, as Charles Radclyffe suggests in his piece [The Dirty Secret About AI](https://www.forbes.com/sites/charlesradclyffe/2019/06/10/ais-dirty-secret/?sh=10b56de65331), the solution could very well be switching off AI.
+
+If these seem a bit extreme, consider how, with little to no policy changes plastic straws were all but phased out in cafes and coffee shops, as well as a mass decrease in plastic bag sales from supermarkets. There is precident for large change being affected without policy makers enforcing it.
+
+We need to change the personal norms and our degree of excitement with new tech. We need to delay our renewal of our personal IT equipment (even delaying by 6 months the acquisition of our next phone might have an impact), we could shrink the size and number of files that we store in the cloud. 
+
+How do we shift the psychology around this? People feel that they lack agency and believe that they will struggle to make a change on their own. Leadership is needed. This doesn't have to be a policy maker. It just needs to be someone to get the message out about the impacts that our consumer decisions have.
+
+At the end of the day, we would like to say that we tried to save the Earth, even if it's a bit of a lost cause. 
+
+## Summary
+
+In summary, the problems belong to all of us. Whilst bottom up and top down approaches are both necessary, there seems to be a feeling that we are limited in what we can do as individuals. Protesting, voting and citizen action are important, but we also need regulators to get on board, for example: enforcing something like carbon credits.
+
+At the time of writing, the next UK election is in 3 years. In that time most people will have replaced their phones or laptops - why not encourage them to make better choices in that time? Maybe we don't need to "cancel capitalism" - perhaps we can use capitalism to our advantage for what it's good for, providing a free market. 
+
+We can attempt to inscentivise companies to improve their [Environmental, Social, and Governance (ESG) Criteria](https://www.investopedia.com/terms/e/environmental-social-and-governance-esg-criteria.asp).
+
+As engineers and data scientists we can make different decisions - where we train our models, what types of models we make. We've heard before that more data isn't necersarily better for accuracy. We now know it's worse for the environment too. 
```

Comment:
I wonder if it might be nice, in the summary to have a bullet point list of the things we could do in our day to day lives (big and small). Like: 

* Delay replacing your devices for as long as possible.
* Think about where our virtual resources are located.
* Consider the efficiency of our models and code better. 
* Turn off your camera when you can during online meetings ([more here](https://www.sciencedaily.com/releases/2021/01/210114134033.htm))
* Try not to use a virtual background.
* Use energy calculators to see how [your website](https://www.websitecarbon.com/) or [your machine learning model](https://mlco2.github.io/impact/) could be made more efficient.